fn.go

```diff
@@ -1,16 +1,27 @@
 package main
 
 import (
+	"bytes"
 	"context"
+	"fmt"
+	"io"
+
+	"golang.org/x/exp/maps"
```

Comment:
Let's not use this package, from [readme](https://pkg.go.dev/golang.org/x/exp#section-readme):

> Warning: Packages here are experimental and unreliable. Some may one day be promoted to the main repository or other subrepository, or they may be modified arbitrarily or even disappear altogether.